From 29adcf2236042d12fdfab0cb5653e6f1d1cd7c18 Mon Sep 17 00:00:00 2001
From: Cedric Bondier <cedricx.bondier@intel.com>
Date: Mon, 7 Jan 2013 16:15:11 +0100
Subject: [PORT FROM MAIN][BT] Multidevice support: change bluedroid conf and output folder

BZ: 76413

Changing bluedroid configuration for several reasons:
 - Change compilation flags
 - Change output folder to /data/bluedroid
 - Remove deprecated compilation flags

Change-Id: I621a7d705326ea4d0e50d437d15d6d13b8eede8f
Orig-Change-Id: I6e91c1a46bb0c5cc2b010018cd6aaf39da218088
Category: device enablement
Domain: CWS.BT-Common
Origin: internal
Upstream-Candidate: no, not-aosp
Signed-off-by: Raffaele Aquilone <raquiloX>
Reviewed-on: http://android.intel.com:8080/85373
Reviewed-by: Bondier, CedricX <cedricx.bondier@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
---
 Android.mk         |    2 +-
 bta/Android.mk     |    4 ----
 conf/bt_stack.conf |    2 +-
 gki/Android.mk     |    5 -----
 stack/Android.mk   |    5 -----
 5 files changed, 2 insertions(+), 16 deletions(-)

diff --git a/Android.mk b/Android.mk
index b5fe1d8..5f2d274 100644
--- a/Android.mk
+++ b/Android.mk
@@ -13,4 +13,4 @@ include $(call all-subdir-makefiles)
 
 # Cleanup our locals
 bdroid_C_INCLUDES :=
-bdroid_CFLaGS :=
+bdroid_CFLAGS :=
diff --git a/bta/Android.mk b/bta/Android.mk
index f4b8a7e..5eb9517 100644
--- a/bta/Android.mk
+++ b/bta/Android.mk
@@ -4,10 +4,6 @@ LOCAL_PATH:= $(call my-dir)
 
 include $(CLEAR_VARS)
 
-ifeq ($(BOARD_HAVE_BLUETOOTH_BCM),true)
-LOCAL_CFLAGS += \
-	-DBOARD_HAVE_BLUETOOTH_BCM
-endif
 LOCAL_CFLAGS += -DBUILDCFG $(bdroid_CFLAGS)
 
 LOCAL_PRELINK_MODULE:=false
diff --git a/conf/bt_stack.conf b/conf/bt_stack.conf
index 2ca20ea..1a5a7eb 100644
--- a/conf/bt_stack.conf
+++ b/conf/bt_stack.conf
@@ -3,7 +3,7 @@
 BtSnoopLogOutput=false
 
 # BtSnoop log output file
-BtSnoopFileName=/sdcard/btsnoop_hci.log
+BtSnoopFileName=/data/misc/bluedroid/btsnoop_hci.log
 
 # Enable trace level reconfiguration function
 # Must be present before any TRC_ trace level settings
diff --git a/gki/Android.mk b/gki/Android.mk
index 0ece25b..7e60255 100644
--- a/gki/Android.mk
+++ b/gki/Android.mk
@@ -13,11 +13,6 @@ LOCAL_C_INCLUDES:= $(LOCAL_PATH)/common \
 
 LOCAL_CFLAGS += -Werror $(bdroid_CFLAGS)
 
-ifeq ($(BOARD_HAVE_BLUETOOTH_BCM),true)
-LOCAL_CFLAGS += \
-	-DBOARD_HAVE_BLUETOOTH_BCM
-endif
-
 LOCAL_PRELINK_MODULE:=false
 LOCAL_SRC_FILES:= \
     ./ulinux/gki_ulinux.c \
diff --git a/stack/Android.mk b/stack/Android.mk
index 92ce253..3c8fe4e 100755
--- a/stack/Android.mk
+++ b/stack/Android.mk
@@ -34,11 +34,6 @@ LOCAL_C_INCLUDES:= . \
 
 LOCAL_CFLAGS += $(bdroid_CFLAGS)
 
-ifeq ($(BOARD_HAVE_BLUETOOTH_BCM),true)
-LOCAL_CFLAGS += \
-	-DBOARD_HAVE_BLUETOOTH_BCM
-endif
-
 LOCAL_PRELINK_MODULE:=false
 LOCAL_SRC_FILES:= \
     ./a2dp/a2d_api.c \
-- 
1.7.4.1

